services/core-api/app/api/mines/response_models.py

```diff
@@ -109,7 +109,7 @@ def format(self, value):
         'mine_guid': fields.String,
         'document_manager_guid': fields.String,
         'document_name': fields.String,
-        'upload_date': fields.DateTime,
+        'upload_date': fields.String,
         'create_user': fields.String,
         'is_archived': fields.Boolean,
         'archived_date': fields.DateTime,
```

Comment:
is the archived date going to create the same problem?
I wonder if the format `fields.DateTime(dt_format='iso8601')` would also work.

Reply:
Good point.  Found another one in incident document model as well.  `fields.DateTime(dt_format='iso8601')` works too!

Reply:
Nevermind.  fields.DateTime(dt_format='iso8601') actually breaks the test, but str is returning it in ISO format at any rate.